presto-main/src/test/java/com/facebook/presto/execution/TaskTestUtils.java

```diff
@@ -124,7 +125,7 @@ public static LocalExecutionPlanner createTestingPlanner()
         return new LocalExecutionPlanner(
                 metadata,
                 new SqlParser(),
-                new CoefficientBasedCostCalculator(metadata),
+                new LegacyCostCalculator(new CoefficientBasedCostCalculator(metadata)),
```

Comment:
I would split this commit into two. In first I would rename CostCalculator to LegacyCostCalculator, in second I would add new CostCalculator. That way it would be easier to review.